mathics/doc/common_doc.py

```diff
@@ -689,251 +1017,32 @@ def __str__(self):
         return self.test
 
 
+# FIXME: think about - do we need this? Or can we use DjangoMathicsDocumentation and
+# LatTeXMathicsDocumentation only?
```

Comment:
Are we using this? Maybe we could replace it with a function that produces the right class of object.

Reply:
Yes, we are using this. Merging the two is another mess that I don't want to deal with right now. 

If you want to try to do it go head.  I will note that there are still lots of opportunities for DRYing the code and improving it 

The motivation here was simply that we had code in Django to handle Mathics3 Modules and I wanted that used on the LaTeX/PDF side. So moving this code to common code seemed like the way to do this which both accomplishes what I want to do while improving the code. 

However in doing so there have been small breakages here and there precisely because there were two sets of code and an improvement or necessary change was made in one copy but not the other. And the one with the improvement did not make it to the common code. 

Last weekend I had my fill of trying to find and fix these kinds of things. Doing more in order to DRY the code more is left for the future for me.